src/Board.py

```diff
@@ -45,10 +49,48 @@ def displayBoard(board):
         line = chr(x + UNICODE_FOR_A_CHAR)
 
         for i in range(len(board)):
-            
-            if board[x][i]:
+            if board[x][i] == SQUARE_SHIP:
+                line += " S "
+            elif board[x][i] == SQUARE_FAILED_SHOT:
+                line += " O "
+            elif board[x][i] == SQUARE_SUCCESS_SHOT:
                 line += " X "
             else:
                 line += "   "
 
         print(line)
+
+def shoot(coord, boards):
+    boardsCopy = deepcopy(boards)
+    if (boards["currentPlayer"] == 1):
+        if (shipAt(coord, boards["player2"]["primary"])):
+            boardsCopy["player1"]["opponent"] = updateBoard(boards["player1"]["opponent"], coord, SQUARE_SUCCESS_SHOT)
+            boardsCopy["hit"] = 1
```

Comment:
I'm not sure you need to store `hit` in `boardsCopy`. You can :
`return hit, boardsCopy;`
and then
`hit, newBoards = shoot(coord, boards)`